dev/com.ibm.ws.classloading/src/com/ibm/ws/classloading/internal/AppClassLoader.java

```diff
@@ -339,6 +341,9 @@ boolean removeTransformer(ClassFileTransformer transformer) {
                 Tr.debug(cltc, String.format("%s: [%s] [%s] [%s]", message, getKey(), loc, name));
             }
         }
+        if (hook != null && resourcePath != null && Arrays.equals(bytes, byteResourceInformation.getBytes())) {
+            hook.storeClass(resourcePath, clazz);
```

Comment:
We should measure the startup time effect of this path in the first (cold) run that stores into the SCC to check how much additional overhead it is on the first run.

Reply:
With springAuthServ app and J9-8sr6-recent, there is a SU cost of ~7% and CPU cost of ~4% for 
the SCC integration